AKS/aksRules.json

```diff
@@ -1,4 +1,9 @@
 {
+  "AKS_AmountOfNodesinPools": {
+    "expected": true,
+    "explanation": "The cluster should have at least 3 nodepools for high availability.",
+    "function": "IsPoolMin3Nodes"
+  },
```

Comment:
For clarity, as it is resiliency related, can you please rename the check to it follows the convention SERVICE_DOMAIN_CHECK. Ex: `AKS_Resiliency_MinNodesPerPool`.
Would also be great if you can put this check with the other resiliency-related tests.